src/main/java/org/axonframework/springboot/nativex/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2010-2023. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@NonNullApi
+package org.axonframework.springboot.nativex;
+
+import org.springframework.lang.NonNullApi;
```

Comment:
Can you explain to me what this is for?

Reply:
It's the Spring way of setting Nullable/NonNull annotations. Everything is by default NonNull by this. It's likely irrelevant for this extension, as the implementation won't be called. It is something we might want to take into account for Axon Framework 5. It does seem nice to make everything NonNullby default.